microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,185 @@
+* Changing monitor scale won't potentially cause the app to sporadically hang, when hosting a WebView2 in a VSTO Add-in.  See [Foo].
+-->
 
-* [CoreWebView2CompositionController Class](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller)
-* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createcorewebview2compositioncontrollerasync)
 
-##### [Win32/C++](#tab/win32cpp)
+<!-- ------------------------------ -->
+#### Disadvantages
 
-* [ICoreWebView2CompositionController interface](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller)
-* [ICoreWebView2Environment::CreateCoreWebView2CompositionController method](/microsoft-edge/webview2/reference/win32/icorewebview2environment3#createcorewebview2compositioncontroller)
-
----
+Enabling Window to Visual hosting mode removes support for Windows Shell Handwriting (pen input)<!-- todo: strike "(pen input)"?  do same in overview table --> within the WebView2.
 
 
 <!-- ------------------------------ -->
-#### Output
+#### APIs for Window to Visual hosting
 
-WebView2 can connect its composition tree to `IDCompositionVisual`, `IDCompositionTarget`, or `Windows::UI::Composition::ContainerVisual`.
+For a list of APIs that can be used when configuring WebView2 Window to Visual hosting (or for Windowed hosting), see [Rendering WebView2 in non-framework apps](./overview-features-apis.md#rendering-webview2-in-non-framework-apps) in _Overview of WebView2 features and APIs_.
 
-##### [.NET/C#](#tab/dotnetcsharp)
 
-* [CoreWebView2CompositionController.RootVisualTarget Property](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.rootvisualtarget)
+<!-- ====================================================================== -->
+## Visual hosting: For more granular control over layout
 
-##### [WinRT/C#](#tab/winrtcsharp)
+When using _Visual hosting_, your host app receives spatial input (such as mouse or touch input) from the user, and forwards this input to the WebView2 control.
 
-* [CoreWebView2CompositionController.RootVisualTarget Property](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.rootvisualtarget)
+In Visual hosting, content is embedded at a given location in the application. This location must handle how content will scale and behave when the user interacts with the application.<!-- todo: What content is this referring to, the WebView2 content?  What does it mean to scale when the user interacts with the application - such as a gesture? --> In addition to the window management described for Windowed hosting, Visual hosting requires the app to manage the composition-based rendering when it receives any user interactions.<!-- todo: What does it mean to "manage the ... rendering"? Which user interactions would cause this, or what are the scenarios? -->
```

Comment:
re: first sentence: What content is this referring to, the WebView2 content? What does it mean to scale when the user interacts with the application - such as a gesture?

Reply:
Closing this "Outdated" comment.  
[Fresh Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3271/files#r1777394345)